src/app/data/scenario-event/scenario-event-data.service.ts

```diff
@@ -5,107 +5,84 @@
 */
 
 import { ScenarioEventStore } from './scenario-event.store';
-import { ScenarioEventQuery } from './scenario-event.query';
 import { Injectable } from '@angular/core';
-import { UntypedFormControl } from '@angular/forms';
-import { LegacyPageEvent as PageEvent } from '@angular/material/legacy-paginator';
-import { Router, ActivatedRoute } from '@angular/router';
+import { Sort } from '@angular/material/sort';
 import {
+  Card,
+  DataField,
+  DataFieldType,
   DataValue,
   Move,
   ScenarioEvent,
   ScenarioEventService,
+  User,
 } from 'src/app/generated/blueprint.api';
-import { map, take, tap } from 'rxjs/operators';
-import { BehaviorSubject, Observable, combineLatest } from 'rxjs';
+import { take, tap } from 'rxjs/operators';
 
 export interface DataValuePlus extends DataValue {
+  fieldType: DataFieldType;
+  sortAndFilterValue: (string | number);
   valueArray: string[];
 }
 
 export interface ScenarioEventPlus extends ScenarioEvent {
```

Comment:
I wasn't sure. It was there before but nobody was using the plusDataValues property. I started using it because it was there and then realized that it's not a good idea since values can change separately from events. So caching values on events seemed unwise. If I remove it then I would need to touch several unrelated files.

Reply:
Let's leave it there for now, but add a Jira ticket to remove it later.